kubetest/gke.go

```diff
@@ -437,13 +438,25 @@ func (g *gkeDeployer) IsUp() error {
 	return isUp(g)
 }
 
+func (g *gkeDeployer) dumpLogs() error {
+	cmdLineTokenized := strings.Fields(os.ExpandEnv(*gkeDumpLogsCmd))
+	return control.FinishRunning(exec.Command(cmdLineTokenized[0], cmdLineTokenized[1:]...))
+}
+
 // DumpClusterLogs for GKE generates a small script that wraps
 // log-dump.sh with the appropriate shell-fu to get the cluster
 // dumped.
 //
 // TODO(zmerlynn): This whole path is really gross, but this seemed
 // the least gross hack to get this done.
 func (g *gkeDeployer) DumpClusterLogs(localPath, gcsPath string) error {
+	if *gkeDumpLogsCmd != "" {
+		if err := g.dumpLogs(); err != nil {
+			// Log a warning and carry on
```

Comment:
I think we'd like to throw an error as well - if the additional command fails, then we'd like to see that in the Prow job execution as well.

Reply:
Done.